Tests/fixtures/podcasting.yml

```diff
@@ -0,0 +1,9536 @@
+    version: "14.0.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: Himalaya
+    version: "2.4.41"
+  device:
+    type: phablet
+    brand: Apple
+    model: 'iPhone 7 Plus'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Himalaya/2.6.32 (iPhone; iOS 16.5.1; Scale/3.00; CFNetwork; iPhone13,4)'
+  os:
+    name: iOS
+    version: "16.5.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: Himalaya
+    version: "2.6.32"
+  device:
+    type: phablet
+    brand: Apple
+    model: 'iPhone 12 Pro Max'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Himalaya_test/55 CFNetwork/894 Darwin/17.4.0'
+  os:
+    name: iOS
+    version: "11.2.6"
+    platform: ""
+  client:
+    type: mobile app
+    name: Himalaya
+    version: "55"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'HyperCatcher/1'
+  os: [ ]
+  client:
+    type: mobile app
+    name: HyperCatcher
+    version: "1"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'HyperCatcher/2 CFNetwork/1399 Darwin/22.1.0'
+  os:
+    name: iOS
+    version: "16.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: HyperCatcher
+    version: "2"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'iHeartRadio/9.19.0 (Android 10; SM-G960U Build/QP1A.190711.020)'
+  os:
+    name: Android
+    version: "10"
+    platform: ""
+  client:
+    type: mobile app
+    name: iHeartRadio
+    version: "9.19.0"
+  device:
+    type: smartphone
+    brand: Samsung
+    model: 'Galaxy S9'
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'iHeartPodcasts/1.1.0 (Android 10; SM-A505FN Build/QP1A.190711.020)'
+  os:
+    name: Android
+    version: "10"
+    platform: ""
+  client:
+    type: mobile app
+    name: iHeartRadio
+    version: "1.1.0"
+  device:
+    type: smartphone
+    brand: Samsung
+    model: 'Galaxy A50'
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'iHeartRadio/10.25.1 (iPhone; iOS 16.3.1; iPhone12,1)'
+  os:
+    name: iOS
+    version: "16.3.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: iHeartRadio
+    version: "10.25.1"
+  device:
+    type: smartphone
+    brand: Apple
+    model: 'iPhone 11'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'iHeartRadio/2020052002 CFNetwork/1125.2 Darwin/19.4.0'
+  os:
+    name: iOS
+    version: "13.4"
+    platform: ""
+  client:
+    type: mobile app
+    name: iHeartRadio
+    version: "2020052002"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'IOSAudiobooks/7.4.1.1 CFNetwork/978.0.7 Darwin/18.7.0'
+  os:
+    name: iOS
+    version: "12.4"
+    platform: ""
+  client:
+    type: mobile app
+    name: IOSAudiobooks
```

Comment:
Does this refer to https://apps.apple.com/us/app/audiobooks/id311507490 ? If so, should not the name be "Audiobooks" for this entry and the ones below?

Reply:
You're right, I copied this from opawg, but couldn't figure out which app it might belong to.